features/frontier/config.pan

```diff
@@ -0,0 +1,44 @@
+unique template features/frontier/config;
+
+include 'features/frontier/rpms';
+
+variable FRONTIER_LOCAL_NET ?= list();
+variable FRONTIER_LOCAL_NET = push('127.0.0.1/32');
+variable FRONTIER_HOST_MONITOR ?= list();
+variable FRONTIER_HOST_MONITOR = push('127.0.0.1/32');
+variable FRONTIER_CACHE_MEM ?= "128 MB";
```

Comment:
I did wonder whether this should be a `long` and whether it should be based on system RAM, but it's probably fine as is.